packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -293,7 +288,8 @@ class FuchsiaDevice extends Device {
   bool isSupported() => true;
 
   @override
-  bool supportsRuntimeMode(BuildMode buildMode) => buildMode != BuildMode.jitRelease;
+  bool supportsRuntimeMode(BuildMode buildMode) =>
```

Comment:
Please revert this style only nit per: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo